### DISCUSSION TITLE
best image quality settings

### DISCUSSION THREAD

**Discussion by gtxaspec** [gtxaspec/wz_mini_hacks#80] [General]
Anyone come up with any good settings for maximum video quality?  
you can use `imp_dbg` to change lots of video settings as well as `cmd imp_control`.

**Comment by tensiondriven** [comment]
I had some good dialog going with @gtxaspec in this thread https://github.com/gtxaspec/wz_mini_hacks/issues/62 where I was able to capture raw sensor data, but at last attempt I was not able to decode it in a meaningful way.  (I will look again to see if there's been any updates.)

I didn't bother with tweaking settings/parameters, though that's probably a very good idea.  I was mainly interested in getting the highest quality image off the sensor, with as little compression as possible (ideally none).
